.cargo/config

```diff
@@ -0,0 +1,5 @@
+[build]
+rustflags = ["-C", "link-arg=-undefined", "-C", "link-arg=dynamic_lookup"]
+
+[env]
+CXXFLAGS = "-std=c++11 -Wno-unused-parameter"
```

Comment:
It doesn't compile for me with those settings. I'm on Linux (Debian testing). So maybe those shouldn't be checked in?